src/lib.rs

```diff
@@ -236,7 +242,7 @@ fn build_poi(row: Row) -> Option<Poi> {
         coord: poi_coord,
         poi_type: PoiType {
             id: class.clone(),
```

Comment:
This id should be updated too, to be consistent with the name.  
What about something like `"{class}:{subclass}"` ?